admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,199 @@
+[[admin-guide-cluster-capacity]]
+= Cluster Capacity Analysis
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Monitoring available resources in the cluster is very important as operators can increase
```

Comment:
@bfallonf -- if you can take a second pass at this for readability, i think it would be good.  i think this can be condensed.

it is important to monitor available resources on your cluster in order to ensure future pods are able to schedule.  as a cluster is populated with pods, fragmentation will occur.  pods may only support scheduling on particular sets of nodes based on selection and affinity criteria.  as a result, it can be difficult to get an estimate on how many more pods a cluster can schedule.

the cluster capacity tool simulates a sequence of scheduling decisions to determine how many instances of an input pod may schedule on the cluster before it is exhausted of resources in order to provide a more accurate estimation.